web/source/kmwuitoolbar.ts

```diff
@@ -1003,7 +1010,14 @@ if(!window['keyman']['ui']['name']) {
     {
       if(ui.init && p['HiddenByUser']) ui.oskButtonNode.className = 'kmw_button';
     }
-    osk['addEventListener']('hide',ui.onHideOSK);
+
+
+    ui.registerEvents = function() {
+      let osk = keymanweb.osk;
+
```

Comment:
```suggestion

      if(!osk) {
        return;
      }

```

Other UI `registerEvents` functions test for validity of `osk`.